### PR TITLE
Added wrapping for multiple delivery projects in the Charon table on the projects pages

### PR DESCRIPTION
**Pull request by sofiahag**
As requested by Mattias. See project P25452.

### CODE REVIEW

File: run_dir/static/js/project_samples.js [hunk 1]
```diff
@@ -1179,7 +1179,7 @@ function load_charon_summary(){
           $('#charon-status-failed').text(data['failed']);
           $('#charon-status-runn').text(data['runn']);
           $('#charon-status-hge').text(data['hge']);
-          $('#charon-status-gdp').text(data['gdp'].toString().replace(/\'/," "));
+          $('#charon-status-gdp').text(data['gdp'].toString().replace(/\'/," ").replace(/,/g, ", "));
```
Comment (aanil) [inline run_dir/static/js/project_samples.js:1182]: I think it'll look better with multiple badges than putting everything in one huge badge.

Comment (sofiahag) [reply run_dir/static/js/project_samples.js]: I agree! In that case I think it looks better without commas. I'll put it on stage...

File: run_dir/static/js/project_samples.js [hunk 2]
```diff
@@ -1179,7 +1179,11 @@ function load_charon_summary(){
           $('#charon-status-failed').text(data['failed']);
           $('#charon-status-runn').text(data['runn']);
           $('#charon-status-hge').text(data['hge']);
-          $('#charon-status-gdp').text(data['gdp'].toString().replace(/\'/," "));
+          var del_projs = '';
+          data['gdp'].forEach(function(del_proj){
+              del_projs += '<span class="badge bg-secondary">'+del_proj+'</span>'+' ';
```
Comment (aanil) [inline run_dir/static/js/project_samples.js:1184]: Use padding instead
```suggestion
              del_projs += '<span class="badge bg-secondary p-1">'+del_proj+'</span>';
```
maybe?

File: run_dir/static/css/status_b5.css [hunk 3]
```diff
@@ -432,7 +432,8 @@ pre {
 }
 
 .badge {
-  font-weight:normal;
+  font-weight: normal;
+  white-space: pre-line;
```
Comment (aanil) [inline run_dir/static/css/status_b5.css:436]:
```suggestion
```

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**